pypeit/specobjs.py

```diff
@@ -241,14 +241,19 @@ def get_std(self, multi_spec_det=None):
                 the standard on each detector.
 
         Returns:
-            SpecObj or SpecObjs
+            SpecObj or SpecObjs or None
 
         """
         # Is this MultiSlit or Echelle
         pypeline = (self.PYPELINE)[0]
         if 'MultiSlit' in pypeline:
             # Have to do a loop to extract the counts for all objects
-            SNR = np.median(self.OPT_COUNTS*np.sqrt(self.OPT_COUNTS_IVAR), axis=1)
+            if hasattr(self, 'OPT_COUNTS'):
```

Comment:
A pedantic point, but to me the cleaner logic is:

if OPT_COUNTS
elif BOX_COUNTS
else None.

Reply:
Agreed.  Modified.